buildSrc/src/main/groovy/org/apache/beam/gradle/BeamModulePlugin.groovy

```diff
@@ -681,8 +681,8 @@ class BeamModulePlugin implements Plugin<Project> {
         antlr_runtime                               : "org.antlr:antlr4-runtime:4.7",
         args4j                                      : "args4j:args4j:2.33",
         auto_value_annotations                      : "com.google.auto.value:auto-value-annotations:$autovalue_version",
-        avro                                        : "org.apache.avro:avro:1.11.4",
```

Comment:
we may define an "avro_version" above and use it here, like other dependencies. Due to https://github.com/apache/beam/pull/32770#issuecomment-2532883241 main/test has to be on same version.